### PR TITLE
8288450: Update attribution in gstreamer.md file

### PR DESCRIPTION
**Pull request by sashamatveev**
Doc-only change to update the license attribution in our gstreamer.md and glib.md files.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issues
 * [JDK-8288450](https://bugs.openjdk.org/browse/JDK-8288450): Update attribution in gstreamer.md file
 * [JDK-8288449](https://bugs.openjdk.org/browse/JDK-8288449): Update attribution in glib.md file

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Andy Goryachev](https://openjdk.org/census#angorya) (@andy-goryachev-oracle - Author)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx pull/807/head:pull/807` \
`$ git checkout pull/807`

Update a local copy of the PR: \
`$ git checkout pull/807` \
`$ git pull https://git.openjdk.org/jfx pull/807/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 807`

View PR using the GUI difftool: \
`$ git pr show -t 807`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/807.diff">https://git.openjdk.org/jfx/pull/807.diff</a>

</details>

### CODE REVIEW

**Comment by sashamatveev** [comment]
/solves 8288449: Update attribution in glib.md file

File: modules/javafx.media/src/main/legal/glib.md [hunk 1]
```diff
@@ -28,7 +28,14 @@ code is here:
...
 
-### LGPL 2.1
+### LICENSE
+```
+There are multiple licenses that apply to different parts of Glib. The
+overarching license is LGPL 2.1, however parts are covered by MIT and
+other parts by BSD-style licenses.
+```
+
+#### LGPL 2.1
```
Comment (kevinrushforth) [inline modules/javafx.media/src/main/legal/glib.md:38]: Minor: to match the other sections and also what we did for WebKit, you might change this to `LGPL 2.1 License`

File: modules/javafx.media/src/main/legal/gstreamer.md [hunk 2]
```diff
@@ -26,7 +26,14 @@ code is here:
...
 
-### LGPL 2.1
+### LICENSE
+```
+There are multiple licenses that apply to different parts of GStreamer. The
+overarching license is LGPL 2.1, however parts are covered by BSD-style
+licenses.
+```
+
+#### LGPL 2.1
```
Comment (kevinrushforth) [inline modules/javafx.media/src/main/legal/gstreamer.md:36]: Minor: to match the other sections and also what we did for WebKit, you might change this to `LGPL 2.1 License`

**Review by kevinrushforth — APPROVED** [review]
Looks good with a minor (optional) comment in two places.

**Review by kevinrushforth — APPROVED** [review]

**Comment by sashamatveev** [comment]
8288450: Update attribution in gstreamer.md file [v3]
 - Updated gstreamer.md and glib.md.
 - Removed unused header file (valgrind.h).

**Review by andy-goryachev-oracle — APPROVED** [review]
is this information copied directly from the corresponding source projects?
meaning, when copyright section changes in the future it will be a trivial update?

**Comment by kevinrushforth** [comment]
> is this information copied directly from the corresponding source projects?

Yes.

**Review by kevinrushforth — APPROVED** [review]
Updated version looks good. Thanks.

**Pull request closed without merging**